docs/layouts/shortcodes/generated/forst_configuration.html

```diff
@@ -0,0 +1,120 @@
+            <th class="text-left" style="width: 15%">Default</th>
+            <th class="text-left" style="width: 10%">Type</th>
+            <th class="text-left" style="width: 55%">Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><h5>state.backend.forst.cache.dir</h5></td>
+            <td style="word-wrap: break-word;">(none)</td>
+            <td>String</td>
+            <td>The directory where ForSt caches its SST files, fallback to state.backend.forst.local-dir/cache if not configured.</td>
+        </tr>
+        <tr>
+            <td><h5>state.backend.forst.cache.reserve-size</h5></td>
+            <td style="word-wrap: break-word;">-1</td>
+            <td>Long</td>
+            <td>The reserved size of cache, when set to a positive number, dynamic space checking will be leveraged. This option and the state.backend.forst.cache.size-based-limitoption can be set simultaneously, the smaller cache limit will be used as the upper limit.</td>
+        </tr>
+        <tr>
+            <td><h5>state.backend.forst.cache.size-based-limit</h5></td>
+            <td style="word-wrap: break-word;">-1</td>
+            <td>Long</td>
+            <td>The size-based capacity limit of cache, a non-positive number indicates that there is no limit.</td>
+        </tr>
+        <tr>
+            <td><h5>state.backend.forst.executor.inline-coordinator</h5></td>
+            <td style="word-wrap: break-word;">false</td>
+            <td>Boolean</td>
+            <td>Whether to let the task thread be the coordinator thread responsible for distributing requests.</td>
+        </tr>
+        <tr>
+            <td><h5>state.backend.forst.executor.inline-write</h5></td>
+            <td style="word-wrap: break-word;">true</td>
+            <td>Boolean</td>
+            <td>Whether to let write requests executed within the coordinator thread.</td>
+        </tr>
+        <tr>
+            <td><h5>state.backend.forst.executor.read-io-parallelism</h5></td>
+            <td style="word-wrap: break-word;">3</td>
+            <td>Integer</td>
+            <td>The number of threads used for read IO operations in the executor.</td>
+        </tr>
+        <tr>
+            <td><h5>state.backend.forst.executor.write-io-parallelism</h5></td>
+            <td style="word-wrap: break-word;">1</td>
+            <td>Integer</td>
+            <td>The number of threads used for write IO operations in the executor. Only valid when 'state.backend.forst.executor.inline-write' is false.</td>
+        </tr>
+        <tr>
+            <td><h5>state.backend.forst.local-dir</h5></td>
+            <td style="word-wrap: break-word;">(none)</td>
+            <td>String</td>
+            <td>The local directory (on the TaskManager) where ForSt puts some meta files. Per default, it will be &lt;WORKING_DIR&gt;/tmp. See <code class="highlighter-rouge">process.taskmanager.working-dir</code> for more details.</td>
```

Comment:
is this per task manager?

Reply:
What do you mean by per task manager? All the options are set per job.